plugins/databases/pom.xml

```diff
@@ -81,7 +81,7 @@
         <module>sapdb</module>
         <module>sqlite</module>
         <module>googlebigquery</module>
-		<module>snowflake</module>        
+		<module>snowflake</module> 
```

Comment:
unrelated to this PR remove

Reply:
formatting issue due to unwanted white spaces.